test/fixtures/schema/enum-nullable.json

```diff
@@ -176,7 +176,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"address\": 4\n}"
+                              "content": "{\n  \"address\": 0\n}"
```

Comment:
The value `4` is not being rendered.

```apib
# Group Example
# Resource [/example]

## Action [GET]

+ Response 200 (application/json)

    + Attributes
        + address (enum[number], nullable)
            + 4
```

Reply:
API Elements -> JSON Value is correct. Added #574 to allow for a better first value on type-hinted enums.